components/discourse/discourse.app.js

```diff
@@ -0,0 +1,192 @@
+const get = require("lodash.get");
+const sortBy = require("lodash.sortby");
+const axios = require("axios");
+const { nanoid } = require("nanoid");
+
+module.exports = {
+  type: "app",
+  app: "discourse",
+  propDefinitions: {
+    categories: {
+      type: "string[]",
+      label: "Categories",
+      description:
+        "The Discourse categories you want to watch for changes. **Leave blank to watch all categories**.",
+      async options() {
+        // Categories endpoint does not implement pagination
```

Comment:
You should do the only first query does work then -- ie. https://github.com/PipedreamHQ/pipedream/blob/master/components/ringcentral/sources/new-event/new-event.js#L32

Reply:
good call, fixed